src/components/Atoms/Button/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import {
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+  styled,
+} from '@mui/material'
+interface iProps extends MuiButtonProps {}
+const CustomizedButton = styled(MuiButton)(() => ({}))
+const CutomButton: React.FC<iProps> = (props) => {
+  const { children } = props
+  return (
+    <>
```

Comment:
fragment wrapping not required for single component.

Reply:
removed